Build.proj

```diff
@@ -12,6 +12,7 @@
     <ProjectToBuild Include="$(MSBuildThisFileDirectory)src\*\src\*.*proj" />
 
     <!-- Build all Test Projects -->
+    <TestExclusions Condition="'$(OS)' != 'Windows_NT'" Include="$(MSBuildThisFileDirectory)src\Microsoft.Bcl.HashCode\tests\Microsoft.Bcl.HashCode.Tests.csproj" /> <!-- This test is .NET Framework only -->
```

Comment:
This needs to go directly into the test project now. See https://github.com/dotnet/maintenance-packages/blob/8aa876ae7ba097a9f0eec4dbb9c5d55ce9c02e2d/src/System.Buffers/tests/System.Buffers.Tests.csproj#L6-L7 for an example.